tests/test_atom_client.py

```diff
@@ -0,0 +1,335 @@
+# -*- coding: UTF-8 -*-
+import os
+import pytest
```

Comment:
I recently discovered [isort](https://github.com/timothycrosley/isort) which is a tool to sort imports, grouping according to stdlib vs not and sorting alphabetically. Maybe we can start using it?